packages/fast-usdc/src/typeGuards.js

```diff
@@ -37,3 +37,8 @@ export const PendingTxShape = {
   status: M.or(...Object.values(PendingTxStatus)),
 };
 harden(PendingTxShape);
+
+export const QueryParamsShape = {
```

Comment:
this is too generic and is easily confused with `QueryParamsShape` in address.js

consider `EudParamShape`